bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/internal/clangd/ClangdFallbackManager.java

```diff
@@ -42,11 +42,11 @@ public final class ClangdFallbackManager implements ClangdFallbackFlags {
 	class FallbackFlags {
 		// fallbackFlags is used as element name in the initialize jsonrpc call:
 		// "initializationOptions":{"fallbackFlags":[...
-		final String fallbackFlags[];
+		String fallbackFlags[];
```

Comment:
Can we restore final modifier here? Immutable object is usually much better than mutable one.

Reply:
I know. Unfortunately is the size of this field depending on the  `systemIncludes` legnth.

Reply:
>  the size of this field depending on the systemIncludes legnth.

true. But, since we modify this field in constructor only, do you think we can add "else" branch there and initialize this field for `null` `systemIncludes` as well?

Reply:
Sure. Sorry :-/

Reply:
No problem. Thank you.